_content/gateways/thethingsindoor/faq.md

```diff
@@ -11,31 +11,52 @@ This page consolidates some of the most frequently asked questions on [The Thing
 > This section contains the most frequent issues and queries pertaining to the The Things Indoor Gateway. If you find issues that are not covered in this page, feel free to reach out to [The Things Network community forum](https://www.thethingsnetwork.org/forum).
 
 
-**Q. What is the right way to register the gateway in the TTN console?**
+#### Q. What is the right way to register the gateway in the TTN console?
 
 Since this is a EUI based gateway, select the `I'm using the legacy packet forwarder` option while registering the gateway and add your 64bit (8byte) EUI in the `GatewayID` tab. The console automatically prepends lower case `eui-` to your ID. 
 If registering using `ttnctl`, please follow the convention of `eui-<your 64bit EUI in lower case for the hex alphabets>`.
 
-**Can I use any browser to configure the gateway?** 
+#### Q. Can I use any browser to configure the gateway?
 The WiFi config page has a small JS issue when working with Firefox. Chrome will work just fine. There will be an automatic firmware update soon that will fix this.
 
-**Q. I don't live in Europe or in the US, can I still use the gateway?**
+#### Q. I don't live in Europe or in the US, can I still use the gateway?
 
 Currently, the gateway only supports the EU868 and US915 frequency plans. Near the end of Q1, all frequency plans will be supported.
 
-**Q. How do I know if my Gateway is EU868 or US915?**
+#### Q. How do I know if my Gateway is EU868 or US915?
 
 Please have a look at the serial number on the back of your gateway.
 TMBH100**868**xxxxxx -> EU
 TMBH100**915**xxxxxx -> US
 
```

Comment:
Another way to check which version of the gateway you have is to look for the certification mark. EU versions have CE mark while the US versions have the FCC mark.